services/121-service/test/registrations/get-registration.test.ts

```diff
@@ -0,0 +1,227 @@
+import { FspName } from '../../src/fsp/enum/fsp-name.enum';
+import { SeedScript } from '../../src/scripts/seed-script.enum';
+import { ProgramPhase } from '../../src/shared/enum/program-phase.model';
+import { changePhase } from '../helpers/program.helper';
+import {
+  getRegistrations,
+  importRegistrations,
+} from '../helpers/registration.helper';
+import { getAccessToken, resetDB } from '../helpers/utility.helper';
+
+describe('Load PA table', () => {
+  const programId = 3;
+  const referenceId = '63e62864557597e0d';
+  const registration = {
+    referenceId: referenceId,
+    preferredLanguage: 'en',
+    paymentAmountMultiplier: 1,
+    firstName: 'John',
+    lastName: 'Smith',
+    phoneNumber: '14155238886',
+    fspName: FspName.intersolveJumboPhysical,
+    whatsappPhoneNumber: '14155238886',
+    addressStreet: 'Teststraat',
+    addressHouseNumber: '1',
+    addressHouseNumberAddition: '',
+    addressPostalCode: '1234AB',
+    addressCity: 'Stad',
+  };
+
+  const attribute1 = 'whatsappPhoneNumber';
+  const attribute2 = 'addressCity';
+  const attribute3 = 'referenceId'
+
+  const exepectedAttributes = [
```

Comment:
The "shared constant" `nameConstraintQuestionsArray` should be used here, instead of copy-pasting its values...

And the typo should be fixed. ;)

```suggestion
  const expectedAttributes = [
```